src/lock.rs

```diff
@@ -28,7 +28,12 @@ impl TryFrom<&PathBuf> for Lock {
 
     fn try_from(path: &PathBuf) -> Result<Self, Self::Error> {
         let mut config_str = String::new();
-        let mut config_file = File::open(path)?;
+        let mut config_file = match File::open(path) {
```

Comment:
The `?` operator or `.map_err(...)` in case you'd like to extend the error message..

Reply:
Used `with_context` accordingly to anyhow docs, thanks!

Reply:
`with_context` had been broken for a long time. Last time I checked, it prevented the error message from being printed correctly.
Please check, and revert to `.map_err` if the issue is still present.

Reply:
I checked, and everything works in the right way right with `with_context`, probably the issue is already fixed. For example for cleaning we can see the following error:

```shell
$ ./target/debug/zkevm-llvm clean
Error: Unable to remove target LLVM directory

Caused by:
    No such file or directory (os error 2)
```

Reply:
Great, thanks!